packages/calypso-e2e/src/environment.ts

```diff
@@ -0,0 +1,2 @@
+export const COOKIES_PATH =
+	process.env.COOKIES_PATH !== null ? ( process.env.COOKIES_PATH as string ) : '';
```

Comment:
I think this can be simplified by using the nullish operator
```suggestion
export const COOKIES_PATH = process.env.COOKIES_PATH ?? '';
```